comictalker/comiccacher.py

```diff
@@ -117,6 +121,13 @@ def create_cache_db(self) -> None:
                 + "credits TEXT,"  # JSON: "{"name": "Bob Shakespeare", "role": "Writer"}"
                 + "teams TEXT,"  # Newline separated
                 + "story_arcs TEXT,"  # Newline separated
+                + "genres TEXT,"  # Newline separated
+                + "tags TEXT,"  # Newline separated
+                + "critical_rating FLOAT,"
+                + "manga TEXT,"  # Yes/YesAndRightToLeft/No
+                + "maturity_rating TEXT,"
+                + "language TEXT,"  # 2-letter iso code
+                + "country TEXT,"  # 2-letter iso code
```

Comment:
I don't intend to enforce the format of these. The only enforcement will be data types.